### PR TITLE
Chrysalis Docs - Style Guide and Diataxis Review II

### PR DESCRIPTION
**Pull request by charlesthompson3**
Documentation reviewed for style guide adherance and diataxis updates for the following sections:

- Libraries (Overview, Wallet, and Client)
- Guides (Developer)

### CODE REVIEW

**Comment by charlesthompson3** [comment]
@iotaledger/tech-writers, carve out time for the Developer Guide. It takes awhile.

File: docs/libraries/client.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-description: The wallet library is a stateful library for developers, recommended if using IOTA for token transfers. This library has a stateful design and adds support for handling and monitoring of multiple account/addresses.
+description: The wallet library is a stateful library for developers, recommended if using IOTA for token transfers. This library has a stateful design and adds support for the handling and monitoring of multiple account/addresses.
```
Comment (lucas-tortora) [inline docs/libraries/client.md:2]:
```suggestion
description: The wallet library is a stateful library for developers, recommended if you want to use IOTA for token transfers. This library has a stateful design and supports handling and monitoring multiple accounts and addresses.
```

File: docs/libraries/client.md [hunk 2]
```diff
@@ -20,7 +21,7 @@ The official client library for interacting with the IOTA Tangle allows you to:
 - Generate addresses.
 - Interact with an IOTA node.
 
-If you mainly intend to process value transfers, we recommend you use our stateful [wallet library](wallet.md) instead.
+However, if you are mainly processing value transfers, you should use our stateful [wallet library](wallet.md) instead.
```
Comment (lucas-tortora) [inline docs/libraries/client.md:24]:
```suggestion
However, if you want to process value transfers, you should use our stateful [wallet library](wallet.md) instead.
```

File: docs/libraries/overview.md [hunk 3]
```diff
@@ -30,16 +31,15 @@ The official IOTA libraries serve as `one-source-code-of-truth` to IOTA users an
...
+The IOTA foundation is in charge of maintaining the official client libraries. You can find the libraries' source code in the [Official GitHub repository](https://github.com/iotaledger).
 
 
 ## Utilities
 
 #### **Developer tools** ####    
-- [Official explorer](https://explorer.iota.org/chrysalis)
-- [Devnet faucet](https://faucet.devnet.chrysalis2.com/)
-- [Commandline wallet](https://github.com/iotaledger/cli-wallet)
+- [Devnet faucet](https://faucet.devnet.chrysalis2.com/).
```
Comment (lucas-tortora) [inline docs/libraries/overview.md:42]:
```suggestion
- [Official explorer](https://explorer.iota.org/mainnet).
- [Devnet faucet](https://faucet.devnet.chrysalis2.com/).
```

File: docs/libraries/wallet.md [hunk 4]
```diff
@@ -8,15 +8,14 @@ keywords:
 - nodejs
 - python
 - client
+- reference
 ---
 
 # IOTA Wallet Library
 
-The wallet library is a stateful library for developers; essentially it's a programmable wallet. We recommended that you use this library if you are mainly using IOTA for token transfers. This library stateful design add support for handling and monitoring of multiple account/addresses.
+The wallet library is a stateful library for developers; essentially, it's a programmable wallet. However, if you are mainly using IOTA to transfer tokens, you should use this library as its stateful design adds support for the handling and monitoring of multiple accounts/addresses.
```
Comment (lucas-tortora) [inline docs/libraries/wallet.md:16]:
```suggestion
The wallet library is a stateful library for developers; essentially, it's a programmable wallet. However, if you use IOTA to transfer tokens, you should use this library as its stateful design supports handling and monitoring multiple accounts and addresses.
```

File: docs/libraries/wallet.md [hunk 5]
```diff
@@ -8,15 +8,14 @@ keywords:
...
 
-Additionally, it also supports the [Stronghold.rs](https://github.com/iotaledger/stronghold.rs) enclave, incorporating the best security practices in one package:
-
-[IOTA Wallet Library full documentation](https://wiki.iota.org/wallet.rs/welcome).
+Additionally, it also supports the [Stronghold.rs](https://github.com/iotaledger/stronghold.rs) enclave, incorporating the best security practices in one package. You can read more about the [IOTA Wallet Library documentation in our wiki](https://wiki.iota.org/wallet.rs/welcome).
```
Comment (lucas-tortora) [inline docs/libraries/wallet.md:18]:
```suggestion
It also supports the [Stronghold.rs](https://github.com/iotaledger/stronghold.rs) enclave, incorporating the best security practices in one package. You can read more about the [IOTA Wallet Library documentation in our wiki](https://wiki.iota.org/wallet.rs/welcome).
```

**Review by lucas-tortora — COMMENTED** [review]
LGTME :)

File: docs/guides/overview.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Overview of available guides for the Chrysalis network. Changes from IOTA 1.0 to IOTA 1.5 , aka Chrysalis.  
+description: An overview of the available guides for the Chrysalis network that also outlines the Changes from IOTA 1.0 to IOTA 1.5, aka Chrysalis.
```
Comment (aleksei-korolev) [inline docs/guides/overview.md:2]:
```suggestion
description: An overview of the guides for the Chrysalis network that also outlines the changes from IOTA 1.0 to IOTA 1.5 Chrysalis.  
```

File: docs/libraries/client.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-description: The wallet library is a stateful library for developers, recommended if using IOTA for token transfers. This library has a stateful design and adds support for handling and monitoring of multiple account/addresses.
+description: The wallet library is a stateful library for developers and it is recommended if you want to use IOTA for token transfers. This library has a stateful design and supports the handling and monitoring of multiple accounts and addresses.
```
Comment (aleksei-korolev) [inline docs/libraries/client.md:2]:
```suggestion
description: The wallet library is a stateful library for developers who want to use IOTA for token transfers. This library has a stateful design and supports the handling and monitoring of multiple accounts and addresses.
```

**Comment by charlesthompson3** [comment]
@iotaledger/tech-writers final review👌

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**